cpp/net/url_fetcher_test.cc

```diff
@@ -7,6 +9,9 @@
 #include <sys/stat.h>
 #include <sys/wait.h>
 #include <unistd.h>
+#ifdef HAVE_VFORK_H
```

Comment:
I think vfork.h might not exist and on some platforms the vfork API exists but does not work correctly. These days it shouldn't be necessary to use vfork but it fixed this test on my laptop.